src/main/com/yetanalytics/lrs/pedestal/openapi.cljc

```diff
@@ -0,0 +1,233 @@
+     :responses {204 (g/response "No content" )}
+     :operationId :post-state
+     :description  "Stores or changes the document specified by the given  stateId that exists in the context of the specified Activity, Agent, and registration (if specified)."}
+    :xapi.activities.state.DELETE.request/params
+    {:params    {:activityId :r#IRI
+                 :agent :r#Agent
+                 :?#registration :r#UUID
+                 :?#stateId :t#string}
+     :responses  {204 (g/response "No content" )}
+     :operationId      :delete-state
+     :description     "Deletes all documents associated with the specified Activity, Agent, and registration (if specified), or just the document specified by stateId"}
+    :xapi.agents.profile.GET.request/params
+    {:params  {:agent :r#Agent
+               :?#profileId :t#string
+               :?#since :r#Timestamp}
+     :responses {200 (g/response "If profileId is included in the request, the specified document.  Otherwise, an array of profileId for the specified Agent.")}
+     :operationId :get-agents-profile
+     :description  "Fetches the specified Profile document in the context of the specified Agent.  The semantics of the request are driven by the \"profileId\" parameter. If it is included, the GET method will act upon a single defined document identified by \"profileId\". Otherwise, GET will return the available ids."}
+
+    :xapi.agents.profile.PUT.request/params
+    {:params {:agent :r#Agent
+              :profileId :t#string}
+     :requestBody (g/request :t#object)
+     :responses {204 (g/response "No content")}
+     :operationId :put-agents-profile
+     :description "Stores or changes the specified Profile document in the context of the specified Agent."}
+    :xapi.agents.profile.POST.request/params
+    {:params {:agent :r#Agent
+              :profileId :t#string}
+     :requestBody (g/request :t#object)
+     :responses  {204 (g/response "No content")}
+     :operationId :post-agents-profile
+     :description "Stores or changes the specified Profile document in the context of the specified Agent."}
+    :xapi.agents.profile.DELETE.request/params
+    {:params {:agent :r#Agent
+              :profileId :t#string}
+     :responses {204 (g/response "No content")}
+     :operationId    :delete-agents-profile
+     :description "Deletes the specified Profile document in the context of the specified Agent."}))
+
+
+
+(def annotations
+  {:health {:operationId :health
+            :responses {200 (gc/response "Empty body---a 200 indicates server is alive")}
+            :description "Simple heartbeat"}
+   :about {:operationId :get-about
+           :description "About info"
+           :responses
+           {200 (gc/response "Object containing body text and optional etag"
+                             (gc/o {:body :t#string
+                                    :#?etag :t#string}))}}
+   :statements-get {:params {:?#statementId :t#string
```

Comment:
not-ideal formatting here, maybe do a pass on this ns with clj-pretty-print or the like

Reply:
aligned